antora-ui-camel/src/css/blog.css

```diff
@@ -106,6 +106,14 @@ article.blog p {
   margin-bottom: 1rem;
 }
 
+.blog .post .post-content .table-wrapper {
```

Comment:
I don't think this is a good approach, doing it this way forces us to add the `<div class="table-wrapper">` in the future.

Reply:
But some of the table are overflowing and putting the table wrapper prevents that. Otherwise the page is unresponsive with the table in it.

Reply:
Would it make more sense to target the `table` element rather than introducing the wrapper?